File: Leetcode/1672/Linear_Search_Algo/main.go
func maximumWealth(accounts [][]int) int {
    var max int = 0
    for _,customer := range accounts {
        var transactions int = 0
        for _,transaction := range customer {
            transactions += transaction
        }
        if (max < transactions) {
            max = transactions
        }
    }
    return max   
}

File: Leetcode/1672/Linear_Search_Algo/main.js
/**
 * @param {number[][]} accounts
 * @return {number}
 */
var maximumWealth = function(accounts) {
    max = 0;
   for (let customer of accounts) {
       let transactions = 0;
       for (let transaction of customer) {
           transactions = transactions + transaction;
       }
       if (max<transactions){
           max = transactions;
       }
   }
   return max;
};